tests/ui/macro_use_imports.rs

```diff
@@ -1,11 +1,40 @@
-// edition:2018
+// compile-flags: --edition 2018
+// aux-build:macro_rules.rs
+// aux-build:macro_use_helper.rs
```

Comment:
What happens if you add `// run-rustfix` her?

Reply:
I get this
```
mod a {
    use mac::{pub_macro, inner_mod_macro, function_macro, ty_macro, pub_in_private_macro}
    use mac;
    use mini_mac::ClippyMiniMacroTest
    use mini_mac;
    use mac::{foofoo::inner, inner::try_err}
    use mac::inner;
    use mac::inner::nested::string_add
    use mac::inner::nested;

    #[derive(ClippyMiniMacroTest)]
    struct Test;

    fn test() {
        pub_macro!();
        inner_mod_macro!();
        pub_in_private_macro!(_var);
        function_macro!();
        let v: ty_macro!() = Vec::default();

        inner::try_err!();
        inner::foofoo!();
        nested::string_add!();
    }
}
```

Reply:
I'm fixing the semi colons right now, and I'll look into why it's duplicating the module import
```
use mac::inner;
    use mac::inner::nested::string_add
```

Reply:
The fix fails to compile with errors about how it can't find the various `mod`s

Reply:
> and I'll look into why it's duplicating the module import

This is because of the span pointing only at `#[macro_use]`. I think this is ok, since we can't know if the import is used other than for the macros.